components/camera/replaypcd/replaypcd.go

```diff
@@ -346,6 +347,19 @@ func (replay *pcdCamera) Stream(ctx context.Context, errHandlers ...gostream.Err
 	return stream, errors.New("Stream is unimplemented")
 }
 
+func (replay *pcdCamera) GetImage(ctx context.Context) (image.Image, func(), error) {
+	stream, err := replay.Stream(ctx)
+	if err != nil {
+		return nil, func() {}, err
+	}
+	defer func() {
+		if err := stream.Close(ctx); err != nil {
+			replay.logger.Errorf("stream failed to close: %w", err)
+		}
+	}()
+	return stream.Next(ctx)
+}
+
```

Comment:
Stream is actually unimplemented in `replaypcd`, you can just return an empty image and an error.